### PR TITLE
Proxy not in use bug

### PR DESCRIPTION
**Pull request by bfday**

### CODE REVIEW

**Comment by revilwang** [comment]
The feature you submitted had been added 13 months ago. Do you checkout from a very old version?

**Comment by revilwang** [comment]
Thanks for the pull request. I suggest reformatting your code to put the left brace on a newline to align the code formatting.

File: src/Io.Gate.GateApi/Client/ApiClient.cs [hunk 1]
```diff
@@ -535,6 +535,10 @@ private async Task<ApiResponse<T>> ExecAsync<T>(RestRequest req, IReadableConfig
                 client.ClientCertificates = configuration.ClientCertificates;
             }
 
+            if (configuration.Proxy != null) {
```
Comment (revilwang) [inline src/Io.Gate.GateApi/Client/ApiClient.cs:538]: put left brace to newline

Comment (bfday) [reply src/Io.Gate.GateApi/Client/ApiClient.cs]: done

**Comment by bfday** [comment]
what's next, what's next?

**Pull request merged**